CardiorespiratoryFitness/CardiorespiratoryFitness/iOS/CRFHeartRateStepViewController.swift

```diff
@@ -113,23 +113,33 @@ public final class CRFHeartRateStepViewController: RSDActiveStepViewController,
         }
     }
     
+    private var _firstAppearance: Bool = true
+    
     public override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         
         (self.countdownDial as! RSDCountdownDial).innerColor = UIColor.clear
+        
+        if _firstAppearance {
+            // Make sure the learn more button is hidden
+            self.learnMoreButton?.isHidden = true
```

Comment:
![IMG_0694](https://user-images.githubusercontent.com/3682103/56305938-288c1080-60f6-11e9-85c4-8d42565208c2.PNG)
![IMG_0695](https://user-images.githubusercontent.com/3682103/56305939-2924a700-60f6-11e9-8b82-f61d7a93c4f1.PNG)

Is this the learn more button you mean here (this is after failing to get an HR after the step test) that should be hidden?

Reply:
@DanWebster With a clean build, this file is packaged.